src/ontology/cl-edit.owl

```diff
@@ -32490,6 +32491,16 @@ AnnotationAssertion(terms:date obo:CL_4047012 "2024-09-06T08:41:39Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047012 "angiogenic pericyte"@en)
 SubClassOf(obo:CL_4047012 obo:CL_0000669)
 
+# Class: obo:CL_4047022 (mesenchymal lymphoid tissue organizer cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31201308") Annotation(oboInOwl:hasDbXref "PMID:34497389") obo:IAO_0000115 obo:CL_4047022 "A stromal cell found in the adult gut that plays a crucial role in the formation and maintenance of gut-associated lymphoid tissues (GALTs). This cell is characterized by it's ability to interact with lymphoid tissue inducer (LTi) cells, express lymphotoxin beta receptor (LTβR) and vascular cell adhesion molecule (VCAM), produce chemokines and cytokines for immune cell recruitment and positioning, and contribute to the structural organization of lymphoid tissues within the intestinal environment. mLTo cells are distinguished from other stromal cells by their specific function in orchestrating the development and maintenance of organized lymphoid structures in the gut, which are essential for intestinal immune responses and homeostasis.")
```

Comment:
"PMID:31201308" not a relevant paper.  Typo?

Reply:
Def reads nicely. Ref still needs fixing so can't check ref for how it supports def.  Would like to close this by end of sprint if poss.